### DISCUSSION TITLE
If it possible to allow all the shorcuts been triggered under diffview buffer

### DISCUSSION THREAD

**Discussion by LintaoAmons** [NeogitOrg/neogit#594] [Q&A]
Like after I view all the changes I have made by diffview, and I want to commit, I just hit `cc` in diffview buffer, then been able to commit

**Comment by CKolkey** [comment]
Because diffview is it's own plugin, you'd have to open an issue over there to allow interop like that. We don't have any say in what goes on there.

**Reply by LintaoAmons** [reply]
Can you give some hint, like which api I should use, etc..., so I can create some custom shortcuts under specific type of buffer?

**Reply by ten3roberts** [reply]
It would be possible without a change to diffview, as their low level api (which we already use) allows for feeding custom bindings, loaders for content etc.

It would not be too difficult to just feed our keybindings into diffview. The issue is conflicts, and as another user has pointed out: overriding user's keybindings in diffview. https://github.com/NeogitOrg/neogit/pull/589

**Reply by milkias17** [reply]
Do you think we should do that by default?
